dev/webpack.config.js

```diff
@@ -13,7 +13,7 @@ const __DEV__ = process.env.NODE_ENV === 'development';
 const clean = arr => arr.filter(item => item !== false);
 
 module.exports = {
-  devtool: __DEV__ ? 'cheap-module-source-map' : 'source-map',
+  devtool: 'source-map',
```

Comment:
It doesn't slow down too much the build time?

Reply:
Is it a blocker @samouss ?

Reply:
Absolutely not!